### PR TITLE
Add support for file-picker using WebUI

### PR DESCRIPTION
**Pull request by joone**
BUG=XWALK-950

### CODE REVIEW

**Comment by PeterWangIntel** [comment]
@joone @kenchris @pozdnyakov 
I'm investigating how to implement File Picker for Crosswalk Linux and has found this patch. I have two questions:
(1) Are there some special reasons that we must implement File Picker by web UI?
     I'm not sure if it breaks our design rule to implement File Picker by libGTK.
(2) Do we still have plan to update this patch?
     Means, if we must implement File Picker by web UI, can I take this patch as base (or reference) to implement File Pciker for Crosswalk Linux?

**Comment by joone** [comment]
@PeterWangIntel  I'm sorry for the late reply.

1) Because Tizen3.0 doesn't support Gtk+. If it is okay to use the Gtk+ file-picker in Crosswalk for  Linux desktop (e.g. Ubuntu, Fedora), you may be able to use Gtk+ file-picker.
2) Yes, I will update the patches. They landed in ozone-wayland so I'm planning to use the file-picker implemented in ozone-wayland.

Thanks!

**Comment by PeterWangIntel** [comment]
@joone

**Comment by PeterWangIntel** [comment]
@joone 
I've found the root reason of problem (1) I mentioned, look at my comment for "FilePickerWebDialog::OnDialogClosed"

**Comment by joone** [comment]
@PeterWangIntel  @kenchris @pozdnyakov Please take a look at the latest CL. There is no dependency on Chromium side any more.

File: runtime/browser/ui/xwalk_web_dialog_view.cc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#include "xwalk/runtime/browser/ui/webui/xwalk_web_contents_handler.h"
+
+namespace xwalk {
+
+// Declared in browser_dialogs.h so that others don't need to depend on our .h.
+// Copied from chrome/browser/ui/views/chrome_web_dialog_view.cc
+gfx::NativeWindow ShowWebDialog(gfx::NativeWindow parent,
+                                content::BrowserContext* context,
+                                ui::WebDialogDelegate* delegate) {
+  views::Widget* widget = NULL;
```
Comment (pozdnyakov) [inline runtime/browser/ui/xwalk_web_dialog_view.cc]: views::Widget* widget = (parent) ? views::Widget::CreateWindowWithParent(view, parent) : views::Widget::CreateWindow(view);

File: runtime/browser/ui/linux_webui/OWNERS [hunk 2]
```diff
@@ -0,0 +1 @@
+joone.hur@intel.com
```
Comment (pozdnyakov) [inline runtime/browser/ui/linux_webui/OWNERS]: well, shouldn't it be proposed on Crosswalk owners ML?

**Comment by pozdnyakov** [comment]
looks good overall

File: runtime/browser/ui/linux_webui/OWNERS (hunk 2 above)
Comment (joone) [inline runtime/browser/ui/linux_webui/OWNERS]: I don't know, but we need the owners file as the project grows

File: runtime/browser/ui/xwalk_web_dialog_view.cc (hunk 1 above)
Comment (joone) [inline runtime/browser/ui/xwalk_web_dialog_view.cc]: Nice comment!

**Comment by joone** [comment]
@pozdnyakov  Can you commit this CL?

**Pull request merged**